src/test/java/it/gov/pagopa/cgn/portal/controller/discount/DiscountApiTest.java

```diff
@@ -468,7 +465,7 @@ void Update_CreateAndUpdateDiscountWithBucket_WithBucketChange_Ok() throws Excep
 
         DiscountEntity discountEntity = TestUtils.createSampleDiscountEntityWithBucketCodes(agreement);
         String firstBucketCodeLoad = discountEntity.getLastBucketCodeLoadUid();
-        azureStorage.uploadCsv(multipartFile.getInputStream(), firstBucketCodeLoad, multipartFile.getSize());
+        azureStorage.uploadCsv(multipartFile.getBytes(), firstBucketCodeLoad, multipartFile.getSize());
```

Comment:
why this change in a PR that should just change descriptive messages with descriptive labels?

Reply:
During this task I noticed a worst-practice, the stream object was created outside the uploadCsv method when it is for its exclusive use so I brought the creation inside leaving the byte array as input.

Reply:
Usually this kind of change should be made inside a related PR, or in small unrelated PRs. This is a big PR that touches 39 files, changes should be easily reviewable. A change like this becomes added cognitive complexity that slows down the review to ask questions.

Let's pass it for now, but remember:
- big PRs should be focused, and if you notice something to clean you should make a new PR
- small PRs allows to follow the boyscout's principle, but remember to report all changes and contexts in description

Reply:
You're right, I'll do that next time.